File: C Programming/Prime_No.c
#include<stdio.h>
#include<conio.h>
int main()
{
    int a,i;
    printf("Enter the Number ");
    scanf("%d",&a);
    for(i=2;i<a;i++)
    {
        if(a%i==0)
        {
            printf("Number is not prime");
            break;
        }
    }
    if(i==a)
    {
        printf("Number is Prime");
    }
    getch();
    return 0;
}

File: Java Programming/Prime_No.java
class Prime_No {

    public static void main(String [] rags)
    {
        int a=5,i;
        System.out.println("The Number is "+a);
        for(i=2;i<a;i++)
        {
            if(a%i==0)
            {
                System.out.println("the Number is not Prime ");
                break;
            }
        }
        if(i==a)
        {
            System.out.println("The Number is Prime ");
        }
    }
}
